doc/clap.txt

```diff
@@ -411,7 +421,7 @@ g:clap_provider_grep_blink                         *g:clap_provider_grep_blink*
 g:clap_provider_grep_opts                          *g:clap_provider_grep_opts*
 
   Type: |String|
-  Default: `''`
+  Default: `'-H --no-heading --vimgrep --smart-case'`
 
   Set this to control the extra arguments that get passed to the grep tool.
   Setting `'--hidden -g "!.git/"'` for example would enable grepping hidden
```

Comment:
Perhaps also change to `'-H --no-heading --vimgrep --smart-case --hidden -g "!.git/"'`?

Reply:
All right, I will adjust it! Maybe, I should add another sentence about the --vimgrep flag?

Reply:
Sure, that would be better.

Reply:
Done!